pkgs/development/python-modules/pyqtwebengine/default.nix

```diff
@@ -51,9 +47,6 @@ in buildPythonPackage (rec {
     qtbase
     qtsvg
     qtwebengine
-  ] ++ lib.optionals (stdenv.buildPlatform != stdenv.hostPlatform) [
-    qtwebchannel
-    qtdeclarative
```

Comment:
I expect here that `qtwebengine` will propagate `qtwebchannel` and `qtdeclarative` and that `qtdeclarative` will propagate itself as a `nativeBuildInput`. But maybe we shouldn't rely on those propagations?